user_guide_src/source/outgoing/view_cells.rst

```diff
@@ -18,11 +26,13 @@ Calling a View Cell
 
 No matter which type of View Cell you are using, you can call it from any view by using the ``view_cell()`` helper function.
 
-The first parameter is the name of the class and method to call, and the second parameter is an array of parameters to pass to the method:
+The first parameter is (1) *the name of the class and method* (Simple Cell) or (2) *the name of the class* (Controlled Cell) to call,
+and the second parameter is an array of parameters to pass to the method:
```

Comment:
This is not quite correct, because in the Controlled Cell, if the method is omitted, we will call the `render` method, but we can also specify a different method (this is optional however).

In both cases, we can also specify an array or string of parameters.

```suggestion
The first parameter is (1) *the name of the class and method* (Simple Cell) or (2) *the name of the class and optional method* (Controlled Cell) to call,
and the second parameter is an array or string of parameters to pass to the method:
```

Reply:
I checked the code. Yes, you are correct.